host/xtest/xtest_main.c

```diff
@@ -117,15 +129,29 @@ int main(int argc, char *argv[])
 
 	xtest_teec_ctx_init();
 
-	if (strcmp(test_suite, "regression") == 0)
-		ret = Do_ADBG_RunSuite(&ADBG_Suite_regression,
-				       argc - optind, argv + optind);
-	else if (strcmp(test_suite, "benchmark") == 0)
-		ret = Do_ADBG_RunSuite(&ADBG_Suite_benchmark,
-				       argc - optind, argv + optind);
-	else {
-		fprintf(stderr, "No test suite found: %s\n", test_suite);
-		ret = -1;
+	for (token = test_suite; ; token = NULL) {
+
+		token = strtok(token, "+");
+		if (!token)
+			break;
+
+		if (strcmp(token, "regression") == 0)
```

Comment:
minor style: could use `!strcmp()` for consistency with the use of `strcmp()` above (around line 90).